doc/client_lua_api.txt

```diff
@@ -30,6 +30,16 @@ Startup
 Mods are loaded during client startup from the mod load paths by running
 the `init.lua` scripts in a shared environment.
 
+In order to load client-side mods in a world, the following conditions need to be satisfied:
+
+1) $path_user/minetest.conf contains the setting `enable_client_modding = true`
+
+2) The client-side mod located in $path_user/clientmods/<modname> is added to
```

Comment:
`<modname>` doesn't render correctly, and I suspect that this is because it is interpreted as a HTML tag. Enclosing all filepaths in backticks (like in [lua_api.txt](https://github.com/minetest/minetest/blob/master/doc/lua_api.txt)) fixes this.